src/pages/workspaces/workspace/Dashboard.js

```diff
@@ -301,19 +312,15 @@ export const WorkspaceDashboard = _.flow(
         div({ style: { fontSize: '1rem', fontWeight: 500, marginBottom: '0.5rem' } }, [
           'Google Bucket'
         ]),
-        div({ style: { display: 'flex' } }, [
-          div({ style: Style.noWrapEllipsis }, [bucketName]),
-          h(Link, {
-            style: { margin: '0 0.5rem', flexShrink: 0 },
-            tooltip: 'Copy bucket name',
-            'aria-label': 'Copy bucket name',
-            onClick: withErrorReporting('Error copying to clipboard', async () => {
-              await clipboard.writeText(bucketName)
-              this.setState({ bucketCopied: true }, () => {
-                setTimeout(() => this.setState({ bucketCopied: undefined }), 1500)
-              })
-            })
-          }, [icon(bucketCopied ? 'check' : 'copy-to-clipboard')])
+        div({ style: { marginBottom: '0.5rem', display: 'flex' } }, [
+          div({ style: { marginRight: '0.5rem', fontWeight: 500 } }, ['Name:']),
+          h(TooltipCell, { tooltip: bucketName, style: { marginRight: '0.5rem', ...Style.noWrapEllipsis } }, [bucketName]),
```

Comment:
Since `TooltipCell` already applies `noWrapEllipsis`, we could omit it here. Also, it will default the tooltip to be identical to the `children`, so we could omit the `tooltip` prop as well.

TOL: `TooltipCell` was originally created to be used in tables, as a way to see the full text that would otherwise be clipped by the cell width. However, there's nothing about its implementation that prevents it from being used outside of that context, and in fact the abstraction fits perfectly well here.